data/flows/order_pizza.yml

```diff
@@ -0,0 +1,100 @@
+flows:
+  order_pizza:
+    name: order pizza
+    description: user asks for a pizza
+    nlu_trigger:
+    - intent:
+        name: inform_pizza
+        confidence_threshold: 0.5
+    steps:
+    - call: "fill_pizza_order"
```

Comment:
You don't need `"` around the id. Let's keep it consistent with the other flows and only use `"` if needed.

Reply:
Updated: https://github.com/RasaHQ/rasa-calm-demo/pull/30/commits/e7001fc9a9d963468e3845fd51afc96a36f0898b